Models/ProtoModel.h

```diff
@@ -110,6 +115,8 @@ class ProtoModel : public QAbstractItemModel {
   bool _dirty;
   Message *_protobuf;
   ProtoModel *_parentModel;
+  static QHash<QString,QHash<Qt::ItemDataRole,QVariant>> _horizontalProtoHeaderData;
+  static QHash<QString,QHash<Qt::ItemDataRole,QVariant>> _verticalProtoHeaderData;
```

Comment:
I was going to use `std::string` as the key type, but QHash doesn't have a hash function for that. If we used `std::string` we wouldn't have to unnecessarily box and unbox the key as a QString, since this is our API and we're the only ones calling it. I would have used `std::unordered_map` but I didn't want to be inconsistent with the existing header data map.